bin/idam-setup.sh

```diff
@@ -37,9 +37,8 @@ curl -s -o /dev/null -XPUT "${HEADERS[@]}" ${IDAM_URI}/services/${CLIENT_ID}/rol
 
 echo "Creating idam users"
 ./bin/idam-create-user.sh citizen,claimant $IDAM_CITIZEN_USERNAME $IDAM_CITIZEN_PASSWORD citizens
-./bin/idam-create-user.sh caseworker,caseworker-divorce $IDAM_CASEWORKER_USERNAME $IDAM_CASEWORKER_PASSWORD caseworker
+./bin/idam-create-user.sh caseworker,caseworker-divorce,caseworker-divorce-courtadmin_beta,caseworker-divorce-systemupdate,caseworker-divorce-courtadmin,caseworker-divorce-bulkscan,caseworker-divorce-superuser,caseworker-divorce-courtadmin-la $IDAM_CASEWORKER_USERNAME $IDAM_CASEWORKER_PASSWORD caseworker
```

Comment:
i believe there is another user with the email casework-divorce-solicitor, is this also needed?

Reply:
There is already a solicitor user `IDAM_TEST_SOLICITOR_USERNAME` configured which is used by test. And as per the latest test we don't need any other solicitor user so if needed we can add it later but not needed as of now.